src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -1,569 +1,2 @@
-                        onFocus={() => this.setState({isFocused: true, highlightedIndex: -1, isUsingKeyboardMovement: false})}
-                        onBlur={() => this.setState({isFocused: false})}
-                        autoCorrect={false}
-                        blurOnSubmit={this.state.filteredEmojis.length > 0}
-                    />
-                </View>
-                {!isFiltered && (
-                    <CategoryShortcutBar
-                        headerEmojis={this.headerEmojis}
-                        onPress={this.scrollToHeader}
-                    />
-                )}
-                <FlatList
-                    ref={(el) => (this.emojiList = el)}
-                    data={this.state.filteredEmojis}
-                    renderItem={this.renderItem}
-                    keyExtractor={this.keyExtractor}
-                    numColumns={CONST.EMOJI_NUM_PER_ROW}
-                    style={[
-                        listStyle,
-                        // This prevents elastic scrolling when scroll reaches the start or end
-                        {overscrollBehaviorY: 'contain'},
-                        // Set overflow to hidden to prevent elastic scrolling when there are not enough contents to scroll in FlatList
-                        {overflowY: this.state.filteredEmojis.length > overflowLimit ? 'auto' : 'hidden'},
-                        // Set scrollPaddingTop to consider sticky headers while scrolling
-                        {scrollPaddingTop: isFiltered ? 0 : CONST.EMOJI_PICKER_ITEM_HEIGHT},
-                    ]}
-                    extraData={[this.state.filteredEmojis, this.state.highlightedIndex, this.props.preferredSkinTone]}
-                    stickyHeaderIndices={this.state.headerIndices}
-                    getItemLayout={this.getItemLayout}
-                    contentContainerStyle={styles.flexGrow1}
-                    ListEmptyComponent={<Text style={[styles.textLabel, styles.colorMuted]}>{this.props.translate('common.noResultsFound')}</Text>}
-                />
-                <EmojiSkinToneList
-                    updatePreferredSkinTone={this.updatePreferredSkinTone}
-                    preferredSkinTone={this.props.preferredSkinTone}
-                />
-            </View>
-        );
-    }
-}
-
-EmojiPickerMenu.propTypes = propTypes;
-EmojiPickerMenu.defaultProps = defaultProps;
-
-export default compose(
-    withWindowDimensions,
-    withLocalize,
-    withOnyx({
-        preferredSkinTone: {
-            key: ONYXKEYS.PREFERRED_EMOJI_SKIN_TONE,
-        },
-        frequentlyUsedEmojis: {
-            key: ONYXKEYS.FREQUENTLY_USED_EMOJIS,
-        },
-    }),
-)(
-    React.forwardRef((props, ref) => (
-        <EmojiPickerMenu
-            // eslint-disable-next-line react/jsx-props-no-spreading
-            {...props}
-            forwardedRef={ref}
-        />
-    )),
-);
+import EmojiPickerMenu from './EmojiPickerMenu';
+export default EmojiPickerMenu;
```

Comment:
Let's please leave this back as it was for now. The build pipeline we have makes it so web will pick up `index.js` and native will pick up `index.native.js`. Doing this adds an unnecessary step and also makes the changes very hard to review in this case.

Reply:
Sorry😅
I renamed all `EmojiPickerMenu*.jsx` back to `index*.js`
(Now there is no barrel importing)

>When I saw `MapView/` directory was barrel-imported like:-
>```
>MapView/
>    |----index.js
>    |----MapView.tsx
>    |----MapView.web.tsx
>
>```
>I thought doing the same with EmojiPickerMenu might be desirable

Reply:
@marcaaron 
Sorry😅
I renamed all `EmojiPickerMenu*.jsx` back to `index*.js`
(Now there is no barrel importing)

>When I saw `MapView/` directory was barrel-imported like:-
>```
>MapView/
>    |----index.js
>    |----MapView.tsx
>    |----MapView.web.tsx
>
>```
>I thought doing the same with EmojiPickerMenu might be desirable

Reply:
Cool let's keep it simple for now. If someone want to do it as part of the TS migration maybe. It's not in scope here. Less is more, thanks!